ngraph_bridge/ngraph_executor.cc

```diff
@@ -473,27 +477,29 @@ NGraphExecutor::InitializeIOTensorPipeline(
         nullptr);
   }
   // Create these pipelined ng tensors only if needed, else reuse from cache
-  size_t num_inputs = ng_exec->get_parameters().size();
-  size_t num_outputs = ng_exec->get_results().size();
-
-  if (num_outputs == 0) {
-    return std::make_pair(
-        errors::Internal("Bad input/output length. Input size: ", num_inputs,
-                         " Output size: ", num_outputs),
-        nullptr);
-  }
+  size_t num_pipelined_inputs = pipelined_input_indexes.size();
```

Comment:
what is difference between ng_exec->get_parameters().size(); and pipelined_input_indexes.size(), if they both are same then why this change?

Reply:
They are not the same
ng_exec->get_parameters().size() returns the total number of inputs for this executable
We are interested in only the ones that are pipelined inputs